src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -44,7 +44,7 @@ export class AuthenticationWrapper implements IAuthenticationWrapper {
     try {
       return await this.getAuthResult([], sessionId);
     } catch (error) {
-      throw error;
+      throw new Error(`${error}`);
```

Comment:
Why is this necessary?

Reply:
So leaving it as is presents an eslint error. This one https://eslint.org/docs/rules/no-useless-catch
I wasn't quite sure what to do because we have quite a number of these try-catch blocks that only throw an error...
Do you have any suggestive fix?

Reply:
You can remove the try...catch statements since they are not really useful here.